test/e2e_awssharedvpc/aws_shared_vpc_test.go

```diff
@@ -0,0 +1,281 @@
+//go:build e2e
+// +build e2e
+
+package e2e_awssharedvpc
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/credentials/stscreds"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/route53"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/openshift/external-dns-operator/pkg/utils"
+	"github.com/openshift/external-dns-operator/test/common"
+
+	configv1 "github.com/openshift/api/config/v1"
+
+	operatorv1beta1 "github.com/openshift/external-dns-operator/api/v1beta1"
+)
+
+const (
+	testNamespace   = "external-dns-test"
+	testServiceName = "test-service"
+	testExtDNSName  = "test-extdns"
+)
+
+var (
+	r53ClientAssumeRole *route53.Route53
+	roleARN             string
+	scheme              *runtime.Scheme
+	hostedZoneID        string
+	hostedZoneDomain    string
+)
+
+func init() {
+	common.InitScheme(scheme)
+}
```

Comment:
Why not calling `initScheme()` in `common`'s `init()` and use `common.Scheme` in this package?

Reply:
Yep that's fair. Done.